### PR TITLE
🚚 update: Add 92nd CatchUp summary

### PR DESCRIPTION
**Pull request by PranavDani**

### CODE REVIEW

File: summary/sessions/092/content.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Duration: 7 hrs 20 mins
+
+==== Topics Discussed
+
+* link:https://twitter.com/mehraramyak[Ramyak Mehra^] shared that he is a part of link:https://summerofcode.withgoogle.com[GSOC^] for current year.
+    ** He talked about his task of adding link:https://docs.microsoft.com/en-us/system-center/vmm/overview?view=sc-vmm-2022[VMM (Virtual Machine Manager)^] support for link:https://www.techtarget.com/whatis/definition/ARM-processor#:~:text=Arm%20processors%20are%20used%20extensively,and%20internet%20of%20things%20devices.[ARM devices^] since currently link:https://www.techopedia.com/definition/20084/x86#:~:text=X86%20is%20the%20term%20used,and%2032%2Dbit%20instruction%20sets.[x86 based devices^] are supported.
+    ** We talked about link:https://www.bittorrent.com[Bittorrent^] parser and ways in which it could be built.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] shared his article about link:https://newsletter.oberai.dev/issues/how-can-devrel-enable-engineering-1270727[DevRel and engineering^].
+    ** He also shared that he will discuss about serverless tech on a talk.
```
Comment (SirusCodes) [inline summary/sessions/092/content.adoc:11]: You can add his meetup link here

File: summary/sessions/092/content.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ** We discussed some features of link:https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui[.NET MAUI^] and some improvements over link:https://dotnet.microsoft.com/en-us/apps/xamarin[Xamarin^].
```
Comment (SirusCodes) [inline summary/sessions/092/content.adoc:12]: Not some improvement. MAUI is successor to Xamarin and it's better

File: summary/sessions/092/content.adoc [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* We talked about a link:https://reactnative.dev[React native^] library which allows updates through the app without requiring app store.
+    ** link:https://flutter.dev[Flutter^] had a similar feature but it was dropped since it was difficult to maintain.
```
Comment (SirusCodes) [inline summary/sessions/092/content.adoc:14]: Here is the discussion link 
https://github.com/flutter/flutter/issues/14330#issuecomment-485565194

File: summary/sessions/092/content.adoc [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    ** We talked about link:https://www.bittorrent.com[Bittorrent^] parser and ways in which it could be built.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] shared his article about link:https://newsletter.oberai.dev/issues/how-can-devrel-enable-engineering-1270727[DevRel and engineering^].
+    ** He also shared that he will discuss about serverless tech on a talk.
+    ** We discussed some features of link:https://docs.microsoft.com/en-us/dotnet/maui/what-is-maui[.NET MAUI^] and some improvements over link:https://dotnet.microsoft.com/en-us/apps/xamarin[Xamarin^].
+* We talked about a link:https://reactnative.dev[React native^] library which allows updates through the app without requiring app store.
```
Comment (SirusCodes) [inline summary/sessions/092/content.adoc:13]: Add link to expo(the lib) https://expo.dev/

File: summary/sessions/092/content.adoc [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] and link:https://twitter.com/rishit_dagli[Rishit Dagli^] talked about visa issues for Canada where they were not receiving their visas, but their parents recieved it.
+* We talked about the learning cure of a developer in a company and how can one recognize where the curve gets flat.
+    ** When things start getting repetitive and redundant, that's when the learning curve gets flat.
+* link:https://twitter.com/SirusTweets[Darshan Rander^] showcased the MVP of his project for sharing music links for all platforms through one link.
+    ** link:https://twitter.com/adityaoberai1[Aditya Oberai^] suggested checking link:https://www.shazam.com[Shazam^] since it provides a similar service.
+* link:https://twitter.com/adityaoberai1[Aditya Oberai^] talked about how a mock design helps in clearly visualising a project and also helps with efficiency.
+* We talked about how Tornado cash leads tp money laundering.
```
Comment (SirusCodes) [inline summary/sessions/092/content.adoc:23]: ~tp~ to

**Review by SirusCodes — CHANGES_REQUESTED** [review]

File: summary/sessions/092/content.adoc (hunk 5 above)
Comment (HarshKapadia2) [reply summary/sessions/092/content.adoc]: Done

File: summary/sessions/092/content.adoc (hunk 4 above)
Comment (HarshKapadia2) [reply summary/sessions/092/content.adoc]: Done

File: summary/sessions/092/content.adoc (hunk 3 above)
Comment (HarshKapadia2) [reply summary/sessions/092/content.adoc]: Done

File: summary/sessions/092/content.adoc (hunk 2 above)
Comment (HarshKapadia2) [reply summary/sessions/092/content.adoc]: Done

**Review by HarshKapadia2 — APPROVED** [review]

**Pull request merged**